control/joystick_commander/src/node.c

```diff
@@ -252,7 +252,12 @@ static int init_joystick( commander_s * commander )
     ret = jstick_init_subsystem();
     
     if( ret == ERROR )
-        printf("init subsystem error\n");    
+        printf("init subsystem error\n");
```

Comment:
We need to enclose the `if` statements with curly brackets, as per Section 12.3 of `CONTRIBUTING.md`.

Same for line 259.